The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/10936814866/job/30361290448) [(plain)](https://github.com/rust-lang/rust/commit/b292b2b1467884f03ae3e8e6fedbba79b5c9bd9e/checks/30361290448/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain

COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---
#13 2.653 Building wheels for collected packages: reuse
#13 2.654   Building wheel for reuse (pyproject.toml): started
#13 2.902   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 2.903   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132715 sha256=dfa09868353292d98f811d3efdb0d54d07389e808efc71d68e3b93c514bf8bec
#13 2.903   Stored in directory: /tmp/pip-ephem-wheel-cache-guouusgw/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#13 2.906 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#13 3.301 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#13 3.302 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#13 3.834 Collecting virtualenv
#13 3.834 Collecting virtualenv
#13 3.889   Downloading virtualenv-20.26.5-py3-none-any.whl (6.0 MB)
#13 4.177      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 6.0/6.0 MB 21.0 MB/s eta 0:00:00
#13 4.223 Collecting distlib<1,>=0.3.7
#13 4.232   Downloading distlib-0.3.8-py2.py3-none-any.whl (468 kB)
#13 4.245      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 42.8 MB/s eta 0:00:00
#13 4.285 Collecting filelock<4,>=3.12.2
#13 4.294   Downloading filelock-3.16.1-py3-none-any.whl (16 kB)
#13 4.331 Collecting platformdirs<5,>=3.9.1
#13 4.340   Downloading platformdirs-4.3.6-py3-none-any.whl (18 kB)
#13 4.419 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 4.607 Successfully installed distlib-0.3.8 filelock-3.16.1 platformdirs-4.3.6 virtualenv-20.26.5
#13 DONE 4.7s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      217024 kB
DirectMap2M:    10268672 kB
DirectMap1G:     8388608 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
    Finished `dev` profile [unoptimized] target(s) in 0.04s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/f8192ba0d00bbbc58a10c45823ba010e98ea4474/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-f8192ba0d00bbbc58a10c45823ba010e98ea4474-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
   Compiling tidy v0.1.0 (/checkout/src/tools/tidy)
    Finished `release` profile [optimized] target(s) in 29.10s
##[endgroup]
fmt check
Diff in /checkout/compiler/rustc_target/src/spec/targets/riscv32_wrs_vxworks.rs:1:
-use crate::spec::{base, StackProbeType, Target, Cc, LinkerFlavor, Lld};
+use crate::spec::{base, Cc, LinkerFlavor, Lld, StackProbeType, Target};
 pub(crate) fn target() -> Target {
     let mut base = base::vxworks::opts();
     base.cpu = "generic-rv32".into();
Diff in /checkout/compiler/rustc_target/src/spec/targets/riscv32_wrs_vxworks.rs:19:
         options: base,
 }
+
Diff in /checkout/compiler/rustc_target/src/spec/targets/riscv64_wrs_vxworks.rs:18:
         options: base,
         options: base,
     }
 }
+
fmt error: Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/compiler/rustc_target/src/spec/targets/aarch64_unknown_uefi.rs" "/checkout/compiler/rustc_target/src/spec/targets/thumbv8m_main_nuttx_eabihf.rs" "/checkout/compiler/rustc_target/src/spec/targets/mips64_unknown_linux_muslabi64.rs" "/checkout/compiler/rustc_target/src/spec/targets/aarch64_unknown_netbsd.rs" "/checkout/compiler/rustc_target/src/spec/targets/aarch64_unknown_linux_gnu_ilp32.rs" "/checkout/compiler/rustc_target/src/spec/targets/x86_64_unknown_linux_none.rs" "/checkout/compiler/rustc_target/src/spec/targets/armv7_unknown_trusty.rs" "/checkout/compiler/rustc_target/src/spec/targets/xtensa_esp32s2_espidf.rs" "/checkout/compiler/rustc_target/src/spec/targets/wasm32_wasi.rs" "/checkout/compiler/rustc_target/src/spec/targets/aarch64_unknown_hermit.rs" "/checkout/compiler/rustc_target/src/spec/targets/mips64el_unknown_linux_gnuabi64.rs" "/checkout/compiler/rustc_target/src/spec/targets/aarch64_be_unknown_linux_gnu_ilp32.rs" "/checkout/compiler/rustc_target/src/spec/targets/x86_64_unknown_hermit.rs" "/checkout/compiler/rustc_target/src/spec/targets/riscv32imc_esp_espidf.rs" "/checkout/compiler/rustc_target/src/spec/targets/x86_64_unknown_l4re_uclibc.rs" "/checkout/compiler/rustc_target/src/spec/targets/csky_unknown_linux_gnuabiv2.rs" "/checkout/compiler/rustc_target/src/spec/targets/mipsisa64r6el_unknown_linux_gnuabi64.rs" "/checkout/compiler/rustc_target/src/spec/targets/powerpc_unknown_openbsd.rs" "/checkout/compiler/rustc_target/src/spec/targets/aarch64_kmc_solid_asp3.rs" "/checkout/compiler/rustc_target/src/spec/targets/armv5te_unknown_linux_uclibceabi.rs" "/checkout/compiler/rustc_target/src/spec/targets/riscv64gc_unknown_linux_gnu.rs" "/checkout/compiler/rustc_target/src/spec/targets/powerpc_wrs_vxworks.rs" "/checkout/compiler/rustc_target/src/spec/targets/mips64_unknown_linux_gnuabi64.rs" "/checkout/compiler/rustc_target/src/spec/targets/thumbv8m_base_none_eabi.rs" "/checkout/compiler/rustc_target/src/spec/targets/mipsisa32r6_unknown_linux_gnu.rs" "/checkout/compiler/rustc_target/src/spec/targets/riscv64gc_unknown_netbsd.rs" "/checkout/compiler/rustc_target/src/spec/targets/mips_unknown_linux_gnu.rs" "/checkout/compiler/rustc_target/src/spec/targets/powerpc64_unknown_openbsd.rs" "/checkout/compiler/rustc_target/src/spec/targets/s390x_unknown_linux_musl.rs" "/checkout/compiler/rustc_target/src/spec/targets/armv7a_kmc_solid_asp3_eabi.rs" "/checkout/compiler/rustc_target/src/spec/targets/arm_unknown_linux_gnueabi.rs" "/checkout/compiler/rustc_target/src/spec/targets/arm64e_apple_darwin.rs" "/checkout/compiler/rustc_target/src/spec/targets/x86_64_unknown_freebsd.rs" "/checkout/compiler/rustc_target/src/spec/targets/aarch64_pc_windows_msvc.rs" "/checkout/compiler/rustc_target/src/spec/targets/thumbv4t_none_eabi.rs" "/checkout/compiler/rustc_target/src/spec/targets/x86_64_pc_windows_gnullvm.rs" "/checkout/compiler/rustc_target/src/spec/targets/armv7a_none_eabi.rs" "/checkout/compiler/rustc_target/src/spec/targets/loongarch64_unknown_none_softfloat.rs" "/checkout/compiler/rustc_target/src/spec/targets/x86_64_pc_windows_gnu.rs" "/checkout/compiler/rustc_target/src/spec/targets/armv8r_none_eabihf.rs" "/checkout/compiler/rustc_target/src/spec/targets/aarch64_apple_visionos.rs" "/checkout/compiler/rustc_target/src/spec/targets/loongarch64_unknown_none.rs" "/checkout/compiler/rustc_target/src/spec/targets/riscv32imafc_esp_espidf.rs" "/checkout/compiler/rustc_target/src/spec/targets/riscv64_wrs_vxworks.rs" "/checkout/compiler/rustc_target/src/spec/targets/riscv32imac_unknown_none_elf.rs" "/checkout/compiler/rustc_target/src/spec/targets/powerpc64_wrs_vxworks.rs" "/checkout/compiler/rustc_target/src/spec/targets/powerpc_unknown_freebsd.rs" "/checkout/compiler/rustc_target/src/spec/targets/riscv32gc_unknown_linux_musl.rs" "/checkout/compiler/rustc_target/src/spec/targets/x86_64_unknown_linux_gnu.rs" "/checkout/compiler/rustc_target/src/spec/targets/nvptx64_nvidia_cuda.rs" "/checkout/compiler/rustc_target/src/spec/targets/wasm32_wasip2.rs" "/checkout/compiler/rustc_target/src/spec/targets/armv7_unknown_linux_gnueabihf.rs" "/checkout/compiler/rustc_target/src/spec/targets/i586_pc_windows_msvc.rs" "/checkout/compiler/rustc_target/src/spec/targets/arm_unknown_linux_musleabihf.rs" "/checkout/compiler/rustc_target/src/spec/targets/thumbv7em_none_eabihf.rs" "/checkout/compiler/rustc_target/src/spec/targets/x86_64_unknown_hurd_gnu.rs" "/checkout/compiler/rustc_target/src/spec/targets/armv7s_apple_ios.rs" "/checkout/compiler/rustc_target/src/spec/targets/armv7_sony_vita_newlibeabihf.rs" "/checkout/compiler/rustc_target/src/spec/targets/x86_64_pc_solaris.rs" "/checkout/compiler/rustc_target/src/spec/targets/mips64el_unknown_linux_muslabi64.rs" "/checkout/compiler/rustc_target/src/spec/targets/aarch64_unknown_teeos.rs" "/checkout/compiler/rustc_target/src/spec/targets/armv7_unknown_linux_musleabihf.rs" "/checkout/tests/run-make/lto-readonly-lib/lib.rs" "/checkout/tests/run-make/use-suggestions-rust-2018/rmake.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Thu Sep 19 08:03:57 UTC 2024
  network time: Thu, 19 Sep 2024 08:03:57 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/compiler/rustc_target/src/spec/mod.rs
+++ b/compiler/rustc_target/src/spec/mod.rs
@@ -1842,6 +1842,8 @@ supported_targets! {
     ("powerpc-wrs-vxworks", powerpc_wrs_vxworks),
     ("powerpc-wrs-vxworks-spe", powerpc_wrs_vxworks_spe),
     ("powerpc64-wrs-vxworks", powerpc64_wrs_vxworks),
+    ("riscv32-wrs-vxworks", riscv32_wrs_vxworks),
+    ("riscv64-wrs-vxworks", riscv64_wrs_vxworks),
 
     ("aarch64-kmc-solid_asp3", aarch64_kmc_solid_asp3),
     ("armv7a-kmc-solid_asp3-eabi", armv7a_kmc_solid_asp3_eabi),

--- a/compiler/rustc_target/src/spec/targets/aarch64_wrs_vxworks.rs
+++ b/compiler/rustc_target/src/spec/targets/aarch64_wrs_vxworks.rs
@@ -7,7 +7,7 @@ pub(crate) fn target() -> Target {
             description: None,
             tier: Some(3),
             host_tools: Some(false),
-            std: None, // ?
+            std: Some(true),
         },
         pointer_width: 64,
         data_layout: "e-m:e-i8:8:32-i16:16:32-i64:64-i128:128-n32:64-S128-Fn32".into(),

--- a/compiler/rustc_target/src/spec/targets/i686_wrs_vxworks.rs
+++ b/compiler/rustc_target/src/spec/targets/i686_wrs_vxworks.rs
@@ -13,7 +13,7 @@ pub(crate) fn target() -> Target {
             description: None,
             tier: Some(3),
             host_tools: Some(false),
-            std: None, // ?
+            std: Some(true),
         },
         pointer_width: 32,
         data_layout: "e-m:e-p:32:32-p270:32:32-p271:32:32-p272:64:64-\

--- a/compiler/rustc_target/src/spec/targets/powerpc64_wrs_vxworks.rs
+++ b/compiler/rustc_target/src/spec/targets/powerpc64_wrs_vxworks.rs
@@ -14,7 +14,7 @@ pub(crate) fn target() -> Target {
             description: None,
             tier: Some(3),
             host_tools: Some(false),
-            std: None, // ?
+            std: Some(true),
         },
         pointer_width: 64,
         data_layout: "E-m:e-Fi64-i64:64-n32:64-S128-v256:256:256-v512:512:512".into(),

--- a/compiler/rustc_target/src/spec/targets/powerpc_wrs_vxworks.rs
+++ b/compiler/rustc_target/src/spec/targets/powerpc_wrs_vxworks.rs
@@ -13,7 +13,7 @@ pub(crate) fn target() -> Target {
             description: None,
             tier: Some(3),
             host_tools: Some(false),
-            std: None, // ?
+            std: Some(true),
         },
         pointer_width: 32,
         data_layout: "E-m:e-p:32:32-Fn32-i64:64-n32".into(),

--- a/compiler/rustc_target/src/spec/targets/riscv32_wrs_vxworks.rs
+++ b/compiler/rustc_target/src/spec/targets/riscv32_wrs_vxworks.rs
@@ -7,7 +7,7 @@ pub(crate) fn target() -> Target {
             description: None,
             tier: Some(3),
             host_tools: Some(false),
-            std: None, // STD is a work in progress for this target arch
+            std: Some(true),
         },
         pointer_width: 32,
         data_layout: "e-m:e-p:32:32-i64:64-n32-S128".into(),

--- a/compiler/rustc_target/src/spec/targets/riscv32_wrs_vxworks.rs
+++ b/compiler/rustc_target/src/spec/targets/riscv32_wrs_vxworks.rs
@@ -1,0 +1,24 @@
+use crate::spec::{base, StackProbeType, Target, TargetOptions};
+
+pub(crate) fn target() -> Target {
+    Target {
+        llvm_target: "riscv32".into(),
+        metadata: crate::spec::TargetMetadata {
+            description: None,
+            tier: Some(3),
+            host_tools: Some(false),
+            std: None, // STD is a work in progress for this target arch
+        },
+        pointer_width: 32,
+        data_layout: "e-m:e-p:32:32-i64:64-n32-S128".into(),
+        arch: "riscv32".into(),
+        options: TargetOptions {
+            cpu: "generic-rv32".into(),
+            llvm_abiname: "ilp32d".into(),
+            max_atomic_width: Some(32),
+            features: "+m,+a,+f,+d,+c,+zicsr".into(),
+            stack_probes: StackProbeType::Inline,
+            ..base::vxworks::opts()
+        },
+    }
+}

--- a/compiler/rustc_target/src/spec/targets/riscv64_wrs_vxworks.rs
+++ b/compiler/rustc_target/src/spec/targets/riscv64_wrs_vxworks.rs
@@ -1,0 +1,24 @@
+use crate::spec::{base, StackProbeType, Target, TargetOptions};
+
+pub(crate) fn target() -> Target {
+    Target {
+        llvm_target: "riscv64".into(),
+        metadata: crate::spec::TargetMetadata {
+            description: None,
+            tier: Some(3),
+            host_tools: Some(false),
+            std: None, // STD is a work in progress for this target arch
+        },
+        pointer_width: 64,
+        data_layout: "e-m:e-p:64:64-i64:64-i128:128-n32:64-S128".into(),
+        arch: "riscv64".into(),
+        options: TargetOptions {
+            cpu: "generic-rv64".into(),
+            llvm_abiname: "lp64d".into(),
+            max_atomic_width: Some(64),
+            features: "+m,+a,+f,+d,+c,+zicsr".into(),
+            stack_probes: StackProbeType::Inline,
+            ..base::vxworks::opts()
+        },
+    }
+}

--- a/compiler/rustc_target/src/spec/targets/riscv64_wrs_vxworks.rs
+++ b/compiler/rustc_target/src/spec/targets/riscv64_wrs_vxworks.rs
@@ -7,7 +7,7 @@ pub(crate) fn target() -> Target {
             description: None,
             tier: Some(3),
             host_tools: Some(false),
-            std: None, // STD is a work in progress for this target arch
+            std: Some(true),
         },
         pointer_width: 64,
         data_layout: "e-m:e-p:64:64-i64:64-i128:128-n32:64-S128".into(),

--- a/compiler/rustc_target/src/spec/targets/x86_64_wrs_vxworks.rs
+++ b/compiler/rustc_target/src/spec/targets/x86_64_wrs_vxworks.rs
@@ -15,7 +15,7 @@ pub(crate) fn target() -> Target {
             description: None,
             tier: Some(3),
             host_tools: Some(false),
-            std: None, // ?
+            std: Some(true),
         },
         pointer_width: 64,
         data_layout:

--- a/library/std/src/sys/alloc/unix.rs
+++ b/library/std/src/sys/alloc/unix.rs
@@ -71,6 +71,7 @@ cfg_if::cfg_if! {
         }
     } else {
         #[inline]
+        #[cfg_attr(target_os = "vxworks", allow(unused_unsafe))]
         unsafe fn aligned_malloc(layout: &Layout) -> *mut u8 {
             let mut out = ptr::null_mut();
             // We prefer posix_memalign over aligned_alloc since it is more widely available, and

--- a/src/doc/rustc/src/platform-support.md
+++ b/src/doc/rustc/src/platform-support.md
@@ -358,12 +358,14 @@ target | std | host | notes
 [`riscv32imc-esp-espidf`](platform-support/esp-idf.md) | ✓ |  | RISC-V ESP-IDF
 [`riscv32imac-esp-espidf`](platform-support/esp-idf.md) | ✓ |  | RISC-V ESP-IDF
 [`riscv32imafc-esp-espidf`](platform-support/esp-idf.md) | ✓ |  | RISC-V ESP-IDF
+[`riscv32-wrs-vxworks`](platform-support/vxworks.md) | ? |  |
 [`riscv64gc-unknown-hermit`](platform-support/hermit.md) | ✓ |   | RISC-V Hermit
 `riscv64gc-unknown-freebsd` |   |   | RISC-V FreeBSD
 `riscv64gc-unknown-fuchsia` |   |   | RISC-V Fuchsia
 [`riscv64gc-unknown-netbsd`](platform-support/netbsd.md) | ✓ | ✓ | RISC-V NetBSD
 [`riscv64gc-unknown-openbsd`](platform-support/openbsd.md) | ✓ | ✓ | OpenBSD/riscv64
 [`riscv64-linux-android`](platform-support/android.md) |   |   | RISC-V 64-bit Android
+[`riscv64-wrs-vxworks`](platform-support/vxworks.md) | ? |  |
 `s390x-unknown-linux-musl` |  |  | S390x Linux (kernel 3.2, musl 1.2.3)
 `sparc-unknown-linux-gnu` | ✓ |  | 32-bit SPARC Linux
 [`sparc-unknown-none-elf`](./platform-support/sparc-unknown-none-elf.md) | * |  | Bare 32-bit SPARC V7+

--- a/src/doc/rustc/src/platform-support.md
+++ b/src/doc/rustc/src/platform-support.md
@@ -358,14 +358,14 @@ target | std | host | notes
 [`riscv32imc-esp-espidf`](platform-support/esp-idf.md) | ✓ |  | RISC-V ESP-IDF
 [`riscv32imac-esp-espidf`](platform-support/esp-idf.md) | ✓ |  | RISC-V ESP-IDF
 [`riscv32imafc-esp-espidf`](platform-support/esp-idf.md) | ✓ |  | RISC-V ESP-IDF
-[`riscv32-wrs-vxworks`](platform-support/vxworks.md) | ? |  |
+[`riscv32-wrs-vxworks`](platform-support/vxworks.md) | ✓ |  |
 [`riscv64gc-unknown-hermit`](platform-support/hermit.md) | ✓ |   | RISC-V Hermit
 `riscv64gc-unknown-freebsd` |   |   | RISC-V FreeBSD
 `riscv64gc-unknown-fuchsia` |   |   | RISC-V Fuchsia
 [`riscv64gc-unknown-netbsd`](platform-support/netbsd.md) | ✓ | ✓ | RISC-V NetBSD
 [`riscv64gc-unknown-openbsd`](platform-support/openbsd.md) | ✓ | ✓ | OpenBSD/riscv64
 [`riscv64-linux-android`](platform-support/android.md) |   |   | RISC-V 64-bit Android
-[`riscv64-wrs-vxworks`](platform-support/vxworks.md) | ? |  |
+[`riscv64-wrs-vxworks`](platform-support/vxworks.md) | ✓ |  |
 `s390x-unknown-linux-musl` |  |  | S390x Linux (kernel 3.2, musl 1.2.3)
 `sparc-unknown-linux-gnu` | ✓ |  | 32-bit SPARC Linux
 [`sparc-unknown-none-elf`](./platform-support/sparc-unknown-none-elf.md) | * |  | Bare 32-bit SPARC V7+

--- a/src/doc/rustc/src/platform-support/vxworks.md
+++ b/src/doc/rustc/src/platform-support/vxworks.md
@@ -14,6 +14,8 @@ Target triplets available:
 - `powerpc-wrs-vxworks`
 - `powerpc64-wrs-vxworks`
 - `powerpc-wrs-vxworks-spe`
+- `riscv32-wrs-vxworks`
+- `riscv64-wrs-vxworks`
 
 ## Target maintainers
 

--- a/tests/assembly/targets/targets-elf.rs
+++ b/tests/assembly/targets/targets-elf.rs
@@ -372,6 +372,9 @@
 //@ revisions: powerpc_wrs_vxworks_spe
 //@ [powerpc_wrs_vxworks_spe] compile-flags: --target powerpc-wrs-vxworks-spe
 //@ [powerpc_wrs_vxworks_spe] needs-llvm-components: powerpc
+//@ revisions: riscv32_wrs_vxworks
+//@ [riscv32_wrs_vxworks] compile-flags: --target riscv32-wrs-vxworks
+//@ [riscv32_wrs_vxworks] needs-llvm-components: riscv
 //@ revisions: riscv32gc_unknown_linux_gnu
 //@ [riscv32gc_unknown_linux_gnu] compile-flags: --target riscv32gc-unknown-linux-gnu
 //@ [riscv32gc_unknown_linux_gnu] needs-llvm-components: riscv
@@ -414,6 +417,9 @@
 //@ revisions: riscv64_linux_android
 //@ [riscv64_linux_android] compile-flags: --target riscv64-linux-android
 //@ [riscv64_linux_android] needs-llvm-components: riscv
+//@ revisions: riscv64_wrs_vxworks
+//@ [riscv64_wrs_vxworks] compile-flags: --target riscv64-wrs-vxworks
+//@ [riscv64_wrs_vxworks] needs-llvm-components: riscv
 //@ revisions: riscv64gc_unknown_freebsd
 //@ [riscv64gc_unknown_freebsd] compile-flags: --target riscv64gc-unknown-freebsd
 //@ [riscv64gc_unknown_freebsd] needs-llvm-components: riscv

